src/Components/Publishing/Sections/ArtworkSaleMessage.tsx

```diff
@@ -36,9 +36,13 @@ export const ArtworkSaleMessage: React.FC<ArtworkSaleMessageQueryResponse> = pro
   }
 
   const saleMessageDisplay = artwork => {
-    const { sale } = artwork
+    const { sale, saleMessage } = artwork
     const isAuction = sale && sale.isAuction
 
+    if (saleMessage === null) {
```

Comment:
#non-blocking Slightly less verbose to do `if (!saleMessage) {` unless there's a reason to explicitly check for `null` here.